content/ja/snippets/others/import_js.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Import a JavaScript library"
+weight: 30
+ie_support: false
+---
+
+JavaScript ライブラリを読み込みます。以下の例では jQuery を CDN から読み込んでいます。
+`<script>` 要素を生成して `<head>` 要素に追加することで JavaScript を読み込めます。
+
+（注意）サードパーティライブラリのご利用は自己責任でお願いします。
```

Comment:
I wrote a caveat with the simplest manner, but it can be more specific like:

- サードパーティライブラリのご利用は自己責任でお願いします。悪意のあるライブラリを使用すると任意の JavaScript コードをサイト内で実行される危険性がありますのでご注意ください。

I'm not sure how specific the caveat should be. Would it better to write a specific way, do you think?

Reply:
I think it's a good idea to have a specific description 👍

Reply:
Improved! [44e874b](https://github.com/autifyhq/autify-javascript-snippets/pull/42/commits/44e874b688cef1c19b22a2df0e92be687a4e577a)